website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Moving Data from Postgres to MotherDuck
+tag: product
+date: 2023/06/22
+description: >-
+  How to continuously stream Postgres data from Postgres to MotherDuck using CloudQuery and Postgres CDC.  
+author: hermanschaaf
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+import { Callout } from 'nextra-theme-docs';
+
+<HowToGuideHeader/>
+
+MotherDuck is a serverless analytics service powered by DuckDB. If you have data in a Postgres database that you'd like to analyze using MotherDuck, this tutorial will show you how to do this using CloudQuery, an open source ELT framework. We will cover:
+ 
+ 1. How to copy a Postgres database into MotherDuck as one-off batch operation, and
+ 2. How to continuously stream Postgres data from Postgres to MotherDuck using Postgres Change Data Capture (CDC).
```

Comment:
Would be good to link to  https://www.cloudquery.io/blog/postgres-cdc-to-any-destination for Postgres CDC.

Reply:
Nice, yeah - added a link 👍